units/src/locktime/absolute.rs

```diff
@@ -12,7 +12,8 @@ use crate::parse::ParseIntError;
 #[cfg(feature = "alloc")]
 use crate::prelude::{Box, String};
 
-/// The Threshold for deciding whether a lock time value is a height or a time (see [Bitcoin Core]).
+/// The Threshold for deciding whether a lock time value is a height or a time
+/// (see [`Bitcoin Core`]).
```

Comment:
I don't think this one is correct. From discussion just recently with @storopoli we are backticking anything that is typically typed as code but "Bitcoin Core" is the name of the project not a code term.

Reply:
Backticks removed